### ISSUE TITLE
publish in chocolatey

### ISSUE THREAD

**Issue by dherman** [notion-cli/notion#13]
We should publish Windows releases with chocolatey.

**Comment by leonheess** [comment]
Please, that'd enable our company to use it.

**Comment by ThatGuyKalvin** [comment]
Published: https://community.chocolatey.org/packages/volta

**Comment by charlespierce** [comment]
That's awesome, thank you @ThatGuyKalvin! Is there an easy path forward for updating the link when there's a new version published?

**Comment by ThatGuyKalvin** [comment]
No worries @charlespierce! I can add you as a maintainer on the chocolatey package. Currently I have to go through a long approval process per version release but I believe you can skip it as you're an official volta maintainer. What's your chocolatey username?

Probably needs automated via a pipeline but it's pretty simple.
https://github.com/ThatGuyKalvin/chocolatey-packages/tree/main/volta
Update version number in volta.nuspec
Update download link and checksum in chocolateyinstall.ps1
Open powershell/cmd from the root package folder.
`choco pack // This creates a .nupkg e.g. volta1.0.8.nupkg
choco install volta -dv -s . -f // This forces a reinstall of volta using the local chocolatey package. Use this to test installer before publishing.
choco push volta1.0.8.nupkg // Publish the package to chocolatey. May require review (3-5 business days)`

Reference: https://docs.chocolatey.org/en-us/create/create-packages

**Comment by leonheess** [comment]
Thank you for looking into this, @charlespierce. This should probably also be mentioned in the docs 🤔

**Comment by ThatGuyKalvin** [comment]
Just checking in @charlespierce - do you have a chocolatey account that I can add as a maintainer?

**Comment by charlespierce** [comment]
@ThatGuyKalvin I don't currently have a chocolatey account, no. Is there a portal I can go through to create one?

**Comment by ThatGuyKalvin** [comment]
@charlespierce yes, you can sign up here https://community.chocolatey.org/account/Register
